test/IntegrationTests/IntegrationTests.csproj

```diff
@@ -1,10 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <!-- No warn is needed for autogenerated code by protobuf: CS8981 - The type name only contains lower-cased ascii characters. -->
-    <NoWarn Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net7.0'">CS8981</NoWarn>
```

Comment:
What changed here with .NET 8 that NoWarn is no longer needed?

Reply:
I think that it was leftover from previous grpc toolking upgrade. In some versions it stopped using internally bad names.